compiler/ballerina-lang/src/test/java/org/ballerinalang/toml/ManifestProcessorTest.java

```diff
@@ -126,44 +129,115 @@ public void testLicenseDescription() throws TomlException {
-        
-        Files.delete(baloPath);
-        Files.delete(tmpDir);
+    public void testSingleDependenciesLinux() throws TomlException, IOException {
+        if (OS.contains("win")) {
+            flag = false;
+        }
+
+        if (flag) {
+            Path tmpDir = Files.createTempDirectory("manifest-test-");
+            Path baloPath = tmpDir.resolve("string_utils.balo");
+            Files.createFile(baloPath);
+
+            // Set tomlData replaced with proper path
+            String tomlData = this.validProjectBlock + "[dependencies] \n string-utils = { path = \""
+                    + baloPath + "\", version = \"1.1.5\" } \n ";
+            Manifest manifest = ManifestProcessor.parseTomlContentFromString(tomlData);
+            Assert.assertEquals(manifest.getDependencies().get(0).getModuleID(), "string-utils");
+            Assert.assertEquals(manifest.getDependencies().get(0).getMetadata().getVersion(), "1.1.5");
+            Assert.assertEquals(manifest.getDependencies().get(0).getMetadata().getPath().toString(),
+                    baloPath.toString());
+
+            Files.delete(baloPath);
+            Files.delete(tmpDir);
+        }
+    }
+
+    // Enabled only on Windows
+    @Test(description = "One dependency added to the dependencies section has an effect")
+    public void testSingleDependenciesWindows() throws TomlException, IOException {
+        if (!OS.contains("win")) {
+            flag = false;
+        }
+
+        if (flag) {
+            Path tmpDir = Files.createTempDirectory("manifest-test-");
+            Path baloPath = tmpDir.resolve("string_utils.balo");
+            Files.createFile(baloPath);
+
+            // Set tomlData replaced with proper path
+            String tomlData = this.validProjectBlock + "[dependencies] \n string-utils = { path = \""
+                    + baloPath.toString().replace("\\", "/") + "\", version = \"1.1.5\" } \n ";
+            Manifest manifest = ManifestProcessor.parseTomlContentFromString(tomlData);
+            Assert.assertEquals(manifest.getDependencies().get(0).getModuleID(), "string-utils");
+            Assert.assertEquals(manifest.getDependencies().get(0).getMetadata().getVersion(), "1.1.5");
+            Assert.assertEquals(manifest.getDependencies().get(0).getMetadata().getPath().toString(),
+                    baloPath.toString());
+
+            Files.delete(baloPath);
+            Files.delete(tmpDir);
+        }
     }
 
+
     @Test(description = "Empty dependency added to the dependencies section has no effect")
     public void testSingleEmptyDependencies() throws TomlException {
         Manifest manifest = ManifestProcessor.parseTomlContentFromString(this.validProjectBlock + "[dependencies] \n " +
                 "string-utils = {} \n");
         Assert.assertEquals(manifest.getDependencies().get(0).getModuleID(), "string-utils");
     }
 
+    // Disabled on Windows
```

Comment:
Shall we change the `description` of the test method appropriately and remove this comment?